config.default.yaml

```diff
@@ -111,51 +111,6 @@ costs: # Costs used in PyPSA-Earth-Sec. Year depends on the wildcard planning_ho
 
 
 industry:
-  St_primary_fraction: 0.9 # fraction of steel produced via primary route versus secondary route (scrap+EAF); today fraction is 0.6
```

Comment:
Sorry for asking post-merge, but don't we need these configs anymore? @hazemakhalek